lib/ansible/plugins/action/reboot.py

```diff
@@ -313,6 +305,14 @@ def run(self, tmp=None, task_vars=None):
             result['elapsed'] = elapsed.seconds
             return result
 
+        post_reboot_delay = int(self._task.args.get('post_reboot_delay', self.DEFAULT_POST_REBOOT_DELAY))
```

Comment:
Since this is now common code and `win_reboot` also accepts `post_reboot_delay_sec`, we also need to loo for that key here.

Reply:
Correct. I would like to get rid of these _sec parameters and plan to deprecate them in a subsequent PR.

Reply:
Agreed.